express-api/src/typeorm/Entities/ProjectAgencyResponse.ts

```diff
@@ -26,7 +26,7 @@ export class ProjectAgencyResponse extends BaseEntity {
   OfferAmount: number;
 
   // Notification Relation
-  @Column({ name: 'NotificationId', type: 'int' })
+  @Column({ name: 'NotificationId', type: 'int', nullable: true })
   NotificationId: number;
```

Comment:
For some reason this column is never used, as I checked in production to confirm there are no references to the Notification Id. Possibly be cause an agency could have more than one notification associated to it.